packages/utils/src/get-top-level-domain.ts

```diff
@@ -0,0 +1,11 @@
+export const getTopLevelDomain =
+  typeof window === 'undefined'
+    ? () => 'localhost'
```

Comment:
Why is it localhost for undefined window?

Reply:
we need some default value for easier types coearsing